react/features/recording/components/LiveStream/AbstractStreamKeyForm.js

```diff
@@ -150,9 +174,43 @@ export default class AbstractStreamKeyForm<P: Props>
      */
     _validateStreamKey(streamKey = '') {
         const trimmedKey = streamKey.trim();
-        const fourGroupsDashSeparated = /^(?:[a-zA-Z0-9]{4}(?:-(?!$)|$)){4}/;
-        const match = fourGroupsDashSeparated.exec(trimmedKey);
+        const match = this.streamLinkRegexp.exec(trimmedKey);
 
         return Boolean(match);
     }
 }
+
+/**
+ * Returns the live streaming props object.
+ *
+ * @param {Object} state - The Redux state.
+ * @returns {LiveStreamingProps}
+ * @private
+ */
+export function _getLiveStreaming(state: Object) {
```

Comment:
Make function called `getLiveStreaming` in functions.any.js actually do this and give you the nice pre-filled object.